docs/devices/SLR2.md

```diff
@@ -28,17 +39,33 @@ description: "Integrate your Hive SLR2 via Zigbee2MQTT with whatever smart home
 
 ### Climate (heat endpoint)
 This climate device supports the following features: `occupied_heating_setpoint`, `local_temperature`, `system_mode`, `running_state`, `pi_heating_demand`.
-- `occupied_heating_setpoint`: Temperature setpoint. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"occupied_heating_setpoint_heat": VALUE}` where `VALUE` is the °C between `7` and `30`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"occupied_heating_setpoint_heat": ""}`.
-- `local_temperature`: Current temperature measured on the device (in °C). To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"local_temperature_heat": ""}`.
-- `system_mode`: Mode of this device. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/set` with payload `{"system_mode_heat": VALUE}` where `VALUE` is one of: `off`, `auto`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"system_mode_heat": ""}`.
-- `running_state`: The current running state. Possible values are: `idle`, `heat`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/get` with payload `{"running_state_heat": ""}`.
+- `occupied_heating_setpoint`: Temperature setpoint. To control publish a message to topic `zigbee2mqtt/FRIENDLY_NAME/heat/set` with payload `{"occupied_heating_setpoint": VALUE}` where `VALUE` is the °C between `7` and `30`. To read send a message to `zigbee2mqtt/FRIENDLY_NAME/heat/get` with payload `{"occupied_heating_setpoint": ""}`.
```

Comment:
Why has this been changed?

Reply:
Hi, the same reason as for water in that the auto-generated endpoint names are incorrect - the endpoint is `zigbee2mqtt/FRIENDLY_NAME/heat/set` for example, with a payload of `{"occupied_heating_setpoint": VALUE}`, not `zigbee2mqtt/FRIENDLY_NAME/set` for example, with a payload of `{"occupied_heating_setpoint_heat": VALUE}` as per auto- generated values.

Reply:
`occupied_heating_setpoint_heat` should also work, but there was a bug which has been fixed recently. This fix is only available in the dev branch  (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html).  (https://github.com/Koenkk/zigbee2mqtt/issues/7470)

Reply:
Hi, the fix to `occupied_heating_setpoint_heat` was the one I spotted and fixed :)    

I hadn't appreciated that both methods work, but I've confirmed that now, ie `zigbee2mqtt/FRIENDLY_NAME/heat/set` with a payload of `{"occupied_heating_setpoint": VALUE}` and `zigbee2mqtt/FRIENDLY_NAME/set` with a payload of `{"occupied_heating_setpoint_heat": VALUE}` both work - sorry for the confusion!  

I'll rework the .md as you suggest.

Reply:
Hi @Koenkk can you please "request changes" to the PR and I'll update?  I can't edit it now its in review state... thanks :)

Reply:
@russdan I don't understand why you cannot edit this, you should be able to push to your [own branch](https://github.com/russdan/zigbee2mqtt.io/tree/patch-1) and this pr will get updated